### DISCUSSION TITLE
Can I use oobabooga windows with my AMD CPU and Nvidia RTX 4070? Thanks!

### DISCUSSION THREAD

**Discussion by Viiincenttt** [oobabooga/text-generation-webui#2016] [Q&A]
I ask this question because the guide says, 

Just download the zip above, extract it, and double-click on "start". The web UI and all its dependencies will be installed in the same folder.

The source codes are here: https://github.com/oobabooga/one-click-installers
There is no need to run the installers as admin.
AMD doesn't work on Windows. <----------------------------------
Huge thanks to [@jllllll](https://github.com/jllllll), [@ClayShoaf](https://github.com/ClayShoaf), and [@xNul](https://github.com/xNul) for their contributions to these installers.

**Comment by Timmytim1976** [comment]
Simple answer: Yes. your only really limited by your VRAM. my unqualified advice would be to stick to LLM Models no bigger then 13B parameters in size and you should be fine. hope this helps

**Reply by Viiincenttt** [reply]
Thanks! I guess the instructions is referring to AMD GPUs instead of CPUs.

**Reply by Timmytim1976** [reply]
I'm just using my laptop to run these models and it's spec's are Amd 5900hx, 32gb ram , RTX 3080 16gb
